examples/galaxies/sdss_dered_10deg2.ecsv

```diff
@@ -0,0 +1,122 @@
+# %ECSV 0.9
+# ---
+# meta:
+#   origin:
+#     citation: Abazajian et al. 2009, ApJS, 182, 543
+#     doi: 10.1088/0067-0049/182/2/543
+#     bibcode: 2009ApJS..182..543A
```

Comment:
if you got this from a query it might be best to just say so

Reply:
I've now stated this in the text